bio-denoise/main.py

```diff
@@ -1,30 +1,75 @@
+from typing import Optional
 from pathlib import Path
 from zipfile import ZipFile
+from io import BytesIO
+
+import logging
+import csv
 
 from coretex import CustomDataset, CustomSample, Experiment, folder_manager
 from coretex.project import initializeProject
 from coretex.bioinformatics import ctx_qiime2
 
 
-def dada2DenoiseSingleSample(sample: CustomSample, denoiseAlgorithm: str, trimLeft: int, truncLen: int, outputDir: Path) -> Path:
+FORWARD_SUMMARY_NAME = "forward-seven-number-summaries.tsv"
+REVERSE_SUMMARY_NAME = "reverse-seven-number-summaries.tsv"
+
+
+def isPairedEnd(demuxSample: CustomSample):
+    demuxTemp = folder_manager.createTempFolder("demux")
```

Comment:
can we add a comment which explains how this function works and what it reads from the qiime metadata?